apps/canvas/canvas.star

```diff
@@ -186,12 +216,19 @@ def get_schema():
         version = "1",
         fields = [
             schema.Text(
-                id = "msg",
-                name = "Canvas API key",
-                desc = "",
-                icon = "compress",
+                id = "api_token",
+                name = "Canvas API Key",
+                desc = "API Key Generated from Canvas",
+                icon = "key",
                 default = "",
             ),
+            schema.Text(
+                id = "domain",
```

Comment:
To protect against abuse, we can unfortunately not accept custom domains or URLs at this moment. We're working on some changes to our infrastructure that are intended to allow this in a relatively near future, but for now I'm afraid we can't merge this.

Reply:
The infrastructure changes I mentioned are now out, so custom domains are now a-ok!